### PR TITLE
[LPTOCPCI-277] Adding firewatch to the OADP scenario

### PR DESCRIPTION
**Pull request by calebevans**

### CODE REVIEW

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-oadp-qe-oadp-qe-automation-main-oadp1.1-ocp4.14-lp-interop-oadp-interop-aws

File: ci-operator/config/oadp-qe/oadp-qe-automation/oadp-qe-oadp-qe-automation-main__oadp1.1-ocp4.13-lp-interop.yaml [hunk 1]
```diff
@@ -56,6 +56,19 @@ tests:
     cluster_profile: aws-cspi-qe
     env:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI"},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
```
Comment (Simranpal) [inline ci-operator/config/oadp-qe/oadp-qe-automation/oadp-qe-oadp-qe-automation-main__oadp1.1-ocp4.13-lp-interop.yaml:62]: I was bit confused about which team/JIRA project some of these steps should be reported to.
For example, 
1) if there is any issue with the operator installation, should LPTOCPCI handle this?
2) Incase of infra issues in aws, should LPTOCPCI still be response?

**Review by Simranpal — APPROVED** [review]
Looks good, just the same comment about where project tickets should be filed, maybe need to discussed with team

File: ci-operator/config/oadp-qe/oadp-qe-automation/oadp-qe-oadp-qe-automation-main__oadp1.1-ocp4.13-lp-interop.yaml (hunk 1 above)
Comment (calebevans) [reply ci-operator/config/oadp-qe/oadp-qe-automation/oadp-qe-oadp-qe-automation-main__oadp1.1-ocp4.13-lp-interop.yaml:62]: Yeah, we should probably discuss this in our next standup together. As long as we are consistent, I think it is fine.

**Pull request merged**